docs/users-manual/file-transfer.rst

```diff
@@ -519,25 +519,70 @@ initial working directory as ``/scratch/test/out1``.
 
     queue
 
+
+.. _dataflow:
+
 Dataflow Jobs
 '''''''''''''
 
 A **dataflow job** is a job that might not need to run because its desired
-outputs already exist. To skip such a job, add the following line to your
-submit file: :index:`dataflow<single: arguments; example>`
+outputs already exist and don't need to be recomputed.
+To skip such a job, add the :subcom:`skip_if_dataflow`
+submit command to your submit file, as in the following example:
+:index:`dataflow<single: arguments; example>`
 
 .. code-block:: condor-submit
 
+    executable      = my_program
+    universe        = vanilla
+
+    error           = logs/err.$(cluster)
+    output          = logs/out.$(cluster)
+    log             = logs/log.$(cluster)
+
+    should_transfer_files = YES
+    when_to_transfer_output = ON_EXIT
+
+    transfer_input_files = in1,in2
+    transfer_output_files = out1
+
+    request_cpus   = 1
+    request_memory = 1024M
+    request_disk   = 10240K
+
     skip_if_dataflow = True
 
-A dataflow job meets any of the following criteria:
+    queue
+
+A dataflow job must meet a number of critera for HTCondor to correctly
+detect if it doesn't need to be run again:
```

Comment:
Should we add a

.. warning::
   If these criteria are not met, and a job is marked as a dataflow job, it might be skipped when it should not be.  HTCondor does not detect nor warn when the criteria are not met.